### PR TITLE
Add build via GHA

### LINKED ISSUE #2: Build for CentOS 8
Build for CentOS 8, to be pushed to UMD.
This should be done via GitHub actions, using the worfklows from https://github.com/EGI-Foundation/bdii

### PR DESCRIPTION
**Pull request by andrea-manzi**
added C7 and C8 build via GHA reusing the bdii workflows

### CODE REVIEW

**Review by gwarf — CHANGES_REQUESTED** [review]
Thanks, looks great! But it would be good to add also the contributing doc cf https://github.com/EGI-Foundation/repository-template

Some other files may be added too (like the Changelog).

**Comment by andrea-manzi** [comment]
i added more files ( CONTRIBUTING, CODEOWNERS, CHANGELOG), i will fill the CHANGELOG file when we release the next version

**Comment by andrea-manzi** [comment]
sorry i did a mistake when adding the CODEOWNERS file, let me fix

File: CODEOWNERS
Comment (gwarf) [inline CODEOWNERS]: You should/could probably add @EGI-Foundation/bdii instead of operations there.

File: CONTRIBUTING.md
Comment (gwarf) [inline CONTRIBUTING.md:85]: Here it should in theory be master as this repos is still relying on master.
Moving to main would be a good idea also, we could plan it as part of this PR (ie updating all references to main), then once merged I can do the necessary changes at the repo level

**Review by gwarf — CHANGES_REQUESTED** [review]
Thanks!
Please check the comments.

File: CODEOWNERS
Comment (gwarf) [inline CODEOWNERS]: Please add back admins, it's always useful to have this, just in case.

**Review by gwarf — CHANGES_REQUESTED** [review]
Great, just one last comment.

File: CODEOWNERS
Comment (andrea-manzi) [reply CODEOWNERS]: sorry..hope it's fine now

**Review by gwarf — APPROVED** [review]
Let's go 🚀 

Thanks!

**Pull request merged**
